cardano-node/src/Cardano/Chairman.hs

```diff
@@ -385,20 +385,21 @@ localInitiatorNetworkApplication sockPath chainsVar securityParam maxBlockNo cha
       (NodeToClientVersionData (nodeNetworkMagic (Proxy @blk) cfg))
       (DictVersion nodeToClientCodecCBORTerm) $ \_peerid ->
 
-    nodeToClientProtocols
-      (InitiatorProtocolOnly $
-         MuxPeer
-           localTxSubmissionTracer
-           localTxSubmissionCodec
-           (localTxSubmissionClientPeer localTxSubmissionClientNull))
-      (InitiatorProtocolOnly $
-         MuxPeer
-           chainSyncTracer
-           (localChainSyncCodec cfg)
-           (chainSyncClientPeer $
-              chainSyncClient chairmanTracer sockPath chainsVar
-                              securityParam maxBlockNo))
-
+    nodeToClientProtocols $
+      NodeToClientProtocols
+        { localChainSyncProtocol = InitiatorProtocolOnly $
+                                     MuxPeer
+                                       localTxSubmissionTracer
+                                       localTxSubmissionCodec
+                                       (localTxSubmissionClientPeer localTxSubmissionClientNull)
+        , localTxSubmissionProtocol = InitiatorProtocolOnly $
+                                        MuxPeer
+                                          chainSyncTracer
+                                          (localChainSyncCodec cfg)
+                                          (chainSyncClientPeer $
+                                             chainSyncClient chairmanTracer sockPath chainsVar
+                                                             securityParam maxBlockNo)
+        }
```

Comment:
These are the wrong way around.

The fact that this does not cause a test failure is deeply worrying. It shows the chairman test must be broken. It cannot connect to the node successfully with this code, but yet the CI is apparently passing.

Reply:
Yep this slipped by me in the update. I hadn't tested that the node works as yet because I wanted to fix the windows build errors before testing the scripts.